src/components/Drawer/index.js

```diff
@@ -85,6 +73,8 @@ export default function Drawer(props) {
         }
     }, [drawerState, isOpen, onOpened]);
 
+    useScrollLock(isOpen);
```

Comment:
Last example in drawer ([https://react-rainbow.io/#!/Drawer/7](https://react-rainbow.io/#!/Drawer/7)) still does not work. Now when open drawer, then birthday modal, close modal and close drawer, scroll remains locked.